config/environments/production.rb

```diff
@@ -96,4 +96,7 @@
   config.webpacker.check_yarn_integrity = false
 
   config.x.policy_disregards_start_date = Date.parse(ENV['POLICY_DISREGARDS_START_DATE'] || '2021-1-8')
+
+  # says whether or not to send alerts to Sentry and support email for undeliverable emails
+  config.x.alert_undeliverable_emails = HostEnv.production?
```

Comment:
Can this not just be `true`?

Reply:
If it's true, then mails from UAT and Staging will be sent as well, as they run in the Rails production environment.  This way, just those running in live production will be sent.